src/main/resources/templates/datewarehouse/etl-jobs-config.json

```diff
@@ -354,17 +861,42 @@
 		"sourcePath" : "log.sagebase.org/certifiedUserPassingRecords/",
 		"tableDescriptor": {
 			"name": "certifiedquizrecords",
-			"columns": {
-				"response_id": "bigint",
-				"user_id": "bigint",
-				"passed": "boolean",
-				"passed_on": "timestamp",
-				"stack": "string",
-				"instance": "string"
+			"description" : "This table contain records of the certification quiz taken by a Synapse user.",
+			"columns":[{
+				"name": "response_id",
+				"type" : "bigint",
+				"comment" : "The unique identifier for the instance of a quiz taken by user."
+			},
+			{
+				"name": "user_id",
+				"type" : "bigint",
+				"comment" : "The identifier of the user who took the quiz."
+			},
+			{
+				"name": "passed",
+				"type" : "boolean",
+				"comment" : "If true, user passed the quiz."
 			},
-			"partitionKeys": {
-				"record_date": "date"
+			{
+				"name": "passed_on",
+				"type" : "timestamp",
+				"comment" : "The date on which the user took or passed the quiz."
```

Comment:
What if they took the quiz but did not pass it?  Is the date populated or null in this field?

Reply:
"The date on which the user took the quiz, regardless of whether user passed or failed the test."